notebooks/AB_testing.py

```
@@ -0,0 +1,231 @@
+        ]
+    ),
+)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ### Call the endpoint
+
+# COMMAND ----------
+
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff (0.7.0)

Undefined name `dbutils`

(F821)